task-docs/init/boilerplate-file-structure.md

```diff
@@ -0,0 +1,30 @@
+This folder structure reflects the current working folder structure for all Origami components, and is what gets generated by `obt init`.
```

Comment:
in a further pr it would be great to have an explanation of what all of these files are :3

Reply:
I'll add it to this one :)